### DISCUSSION TITLE
6top implementations.

### DISCUSSION THREAD

**Discussion by hazelPhd** [contiki-ng/contiki-ng#2079] [Q&A]
I've found a way around this and I wish to delete the discussion.
Thanks.

**Comment by pjonsson** [comment]
There is no `iotlab` platform on the develop branch of Contiki-NG, and the only place where `iotlab` is mentioned in the repository is `examples/benchmarks/result-visualization/README.md`. That file states "The FIT IoT-Lab platform's support must be set up locally.", so there's probably something wrong with the configuration of your local FIT IoT-Lab platform support.
